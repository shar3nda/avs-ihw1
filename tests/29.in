10
2 4 4 1 0 9 0 0 9 6