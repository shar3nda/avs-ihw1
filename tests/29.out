-2 8 -2 -1 10 4 0 1 4 -1 6 9 9 8 4